app/services/data/export/send-to-s3-bucket.service.js

```diff
@@ -5,40 +5,224 @@
  */
-async function _uploadToBucket (params) {
-  const s3Client = new S3Client()
-  const command = new PutObjectCommand(params)
+function _singleUpload (buffer) {
+  const FIVE_MEGA_BYTES = 5242880
 
-  await s3Client.send(command)
+  return buffer.length <= FIVE_MEGA_BYTES
+}
+
+/**
+ * Uploads a file in multiple parts to the specified S3 bucket with the provided key
+ *
+ * @param {Buffer} buffer The file content as a buffer object
+ * @param {String} bucketName Name of the S3 bucket to upload the file to
+ * @param {String} key The path under which the file will be stored in the bucket
+ */
+async function _uploadMultipartFile (bucketName, key, buffer, customConfig) {
+  const s3Client = new S3Client(customConfig)
+  let uploadId
+
+  try {
+    const uploadId = await _createMultipartUpload(s3Client, bucketName, key)
+
+    const uploadResults = await _uploadPartCommand(s3Client, bucketName, key, buffer, uploadId)
+
+    await _completeMultipartUploadCommand(s3Client, bucketName, key, uploadId, uploadResults)
+  } catch (error) {
+    global.GlobalNotifier.omfg('Send to S3 errored', error)
+
+    if (uploadId) {
+      const abortCommand = new AbortMultipartUploadCommand({
+        Bucket: bucketName,
+        Key: key,
+        UploadId: uploadId
+      })
+      await S3Client.send(abortCommand)
+    }
+  }
+}
+
+/**
+ * Uploads a buffer as parts in a multipart upload to an Amazon s3 bucket
+ *
+ * @param {*} s3Client The AWS S3 Client instance used to interact with the service
+ * @param {String} bucketName The name of the bucket where the object will be stored
+ * @param {String} key The key (path) of the object within the bucket
+ * @param {Buffer} buffer The buffer containing the data to be uploaded in parts
+ * @param {String} uploadId The uploadId associated with the created multipart upload
+ *
+ * @returns An array of responses from the upload promises for each part.
+ */
+async function _uploadPartCommand (s3Client, bucketName, key, buffer, uploadId) {
+  const uploadPromises = []
+  // Each part size needs to be a minimum of 5MB to use multipart upload
+  const PART_SIZE = 5242880
+  // Calculating how many parts there will be depending on the size of the buffer
+  const totalParts = Math.ceil(buffer.length / PART_SIZE)
+
+  // Looping through uploading each individual part
+  for (let i = 0; i < totalParts; i++) {
+    const start = i * PART_SIZE
+    const end = Math.min(start + PART_SIZE, buffer.length)
+    uploadPromises.push(
+      s3Client
+        .send(
+          new UploadPartCommand({
+            Bucket: bucketName,
+            Key: key,
+            UploadId: uploadId,
+            Body: buffer.subarray(start, end),
+            PartNumber: i + 1
+          })
+        )
+        .then((response) => {
+          // Once the promise is resolved return the response metadata (this includes the ETag)
+          return response
+        })
+    )
+  }
+
+  return await Promise.all(uploadPromises)
```

Comment:
As `_uploadPartCommand()` is currently being awaited I think we can just do

 ```suggestion
  return Promise.all(uploadPromises)
```